examples/classification.ipynb

```diff
@@ -0,0 +1,942 @@
+{
```

Comment:
<p>it is F<strong>o</strong>rdA, isn't it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>